cmd/primaza/main.go

```diff
@@ -47,6 +47,8 @@ const (
 	EnvHealthCheckInterval         = "HEALTH_CHECK_INTERVAL"
 	DefaultHealthCheckInterval int = 600
 	MinimumHealtCheckInterval  int = 10
+	EnvAppAgent                    = "AGENT_APP"
+	EnvSvcAgent                    = "AGENT_SVC"
```

Comment:
I'd propose to add `Spec` in the name. Also applies in the other files

```suggestion
	EnvAppAgentSpec          = "AGENT_APP_SPEC"
	EnvSvcAgentSpec            = "AGENT_SVC_SPEC"
```

Reply:
Now we are placing th entire deployment in the config map and not just the spec, so may be we shall use the name without `SPEC`

Reply:
good point. What about `Manifest`? I'd really like a more explicit variable name